composer.json

```diff
@@ -13,13 +13,13 @@
     "require": {
         "php": ">=5.3.3",
```

Comment:
I suppose this line should match the PHP version from Symfony.
https://github.com/symfony/symfony/blob/5.0/composer.json#L19
Then Travis will pass.. maybe :)

Reply:
Will try to work in it next week, as soon as I'm back at work.

This was just a quick test to see if it would work, directly from SymfonyCon :)

Reply:
It's okay at is. It should, but not necessarily,  match the PHP version from symfony if this bundle supports only symfony 5.0.